SmartDeviceLink/SDLH264VideoEncoder.m

```diff
@@ -158,6 +158,7 @@ - (BOOL)encodeFrame:(CVImageBufferRef)imageBuffer presentationTimestamp:(CMTime)
     self.currentFrameNumber++;
 
     OSStatus status = VTCompressionSessionEncodeFrame(_compressionSession, imageBuffer, presentationTimestamp, kCMTimeInvalid, NULL, (__bridge void *)self, NULL);
+    VTCompressionSessionCompleteFrames(_compressionSession, presentationTimestamp);
```

Comment:
Alas, trying `VTCompressionSessionCompleteFrames` did not work.

I printed out the `status` returned by `VTCompressionSessionEncodeFrame` in `encodeFrame:presentationTimestamp:` and I got [-12903](https://www.osstatus.com/search/results?platform=all&framework=all&search=-12903). I tried resetting the compression session when I get that error and it fixed the issue (i.e. the background string message shows up as expected).

```
- (BOOL)encodeFrame:(CVImageBufferRef)imageBuffer presentationTimestamp:(CMTime)presentationTimestamp {
    ....

    OSStatus status = VTCompressionSessionEncodeFrame(_compressionSession, imageBuffer, presentationTimestamp, kCMTimeInvalid, NULL, (__bridge void *)self, NULL);
    VTCompressionSessionCompleteFrames(_compressionSession, presentationTimestamp);

    // New code
    if (status == kVTInvalidSessionErr) {
        [self sdl_resetCompressionSession];
    }

    return (status == noErr);
}
```